src/components/Ui/Dropdown.vue

```diff
@@ -5,6 +5,7 @@ const props = defineProps({
   items: Array,
   top: String,
   right: String,
+  subMenuWrapperRight: String,
```

Comment:
This isn't needed, you can just change the value of the `right` prop :D

Reply:
@samuveth The `right` props currently sets the position of the pointer of the dropdown (`sub-menu`'s psuedo element) and if i change the css of the `Dropdown` component directly, it's affecting in other places too, so only I gave a new prop for handling the `right` property of the `sub-menu-wrapper`.

Reply:
Oh right, forgot it was just the pointer..

Reply:
Cool thanks :slightly_smiling_face: